examples/config/time.config

```diff
@@ -81,3 +81,14 @@
   <topic>/world_stats</topic>
 </plugin>
 
+<plugin filename="ShutdownButton">
```

Comment:
Can we create a new config file for it? It's better to keep the configs targeted

Reply:
Done.